Project/src/sysFrame/Object/Model/Mesh/Mesh.cpp

```
@@ -0,0 +1,169 @@
+            assert(0);
+        }
+    }
+
+    return modelData;
+}
+
+Mesh::~Mesh() {
+    commandList_.Reset();
+}
+
+void Mesh::Initialize(const std::string& directory, const std::string& name) {
```

cppcheck (2.10-2)

[style] The function 'Initialize' is never used.

(unusedFunction)